test/e2e/util.go

```diff
@@ -712,3 +724,72 @@ func getSigner(provider string) (ssh.Signer, error) {
 	}
 	return signer, nil
 }
+
+// Retrieve metrics information
+func getMetrics(c *client.Client) (string, error) {
```

Comment:
I think there is a misunderstanding. I'm asking to use getMetrics function in getLatencyMetrics function.